assets/app/views/directives/_status-icon.html

```diff
@@ -1,4 +1,4 @@
-<span ng-switch="status" class="hide-ng-leave status-icon">
+<span ng-switch="status | capitalize" class="hide-ng-leave status-icon">
```

Comment:
Let's not do this. We're not displaying the value. I want to match the exact reason strings we expect.